docs/user_guide/HOWTO/NFS.rst

```diff
@@ -266,8 +266,9 @@ following the instructions in the previous section.
 Server
 ^^^^^^
 
-Create a manifest in the site module. In this example
-the manifest is called nfs_server.pp.
+The server side of the NFS shares should be contained in its own namespace. In
```

Comment:
Should this use the same 'site profile' language as is used in HOWTO/Redundant_LDAP.rst?